architecture/aws/observability.md

```diff
@@ -5,61 +5,62 @@ reviewed: 2024-03-14
 
-- OpenTelemetry-based metrics, which can be collected by enabling OpenTelemetry and exporting the metrics to Amazon CloudWatch
-- Custom metrics with NServiceBus.Metrics which can be exported to Amazon CloudWatch.
+1. OpenTelemetry-based metrics - can be collected by [enabling OpenTelemetry](/nservicebus/operations/opentelemetry.md) and exporting the metrics to Amazon CloudWatch.
+1. Custom metrics with `NServiceBus.Metrics` - can be exported to Amazon CloudWatch - TODO: Link this to TODO2 below.
 
-Try the Amazon CloudWatch sample for NServiceBus metrics and logs →
+Try the Amazon CloudWatch sample for OpenTelemetry → TODO: Link this to TODO1 below.
 
 **TODO**:
 
-1. Create a sample that uses NSB metrics, which integrates with Cloudwatch
+1. Create a sample that uses Otel and integrates with CloudWatch
+1. Create a sample that uses NSB metrics, which integrates with CloudWatch
     1. Collect NSB metrics, send to cloudwatch
     2. Collect [NLog logs with extensions.logging](/samples/logging/extensions-logging/), [send to Cloudwatch](https://docs.aws.amazon.com/prescriptive-guidance/latest/patterns/configure-logging-for-net-applications-in-amazon-cloudwatch-logs-by-using-nlog.html)
 
-#### Amazon X-Ray
+## Amazon X-Ray
 
-[Amazon X-Ray](https://aws.amazon.com/xray/) is a service that can collect trace data from the applications, providing insights that can help identify issues or bottlenecks that could benefit from optimization.
+[Amazon X-Ray](https://aws.amazon.com/xray/) is a service that can receive trace data from applications. The trace data can provide insights that can help identify performance bottlenecks, edge case errors, and other hard to detect issues.
 
 :heavy_plus_sign: Pros:
 
-- A single tool to visualize application-level traces and infrastructure-level traces
-- AWS X-Ray creates a service map using trace data
-- Integrates with other AWS services, such as AWS Lambda, Amazon EC2, Amazon ECS and AWS Elastic Beanstalk
+- Single tool to visualize application-level and infrastructure-level traces
+- Creates a service map using trace data
+- Tightly integrated with other AWS services
 - Fully managed service
 
 :heavy_minus_sign: Cons:
 
-- AWS X-Ray offers a [free tier](https://aws.amazon.com/xray/pricing/), but costs may escalate with increases usage, requiring dedicated monitoring
-- Pricing is based on the amount and the type of telemetry collected
-- AWS X-Ray is designed to work within the AWS ecosystem
-- AWS X-Ray’s `traceId` format differs from the [W3C format](https://www.w3.org/TR/trace-context/#trace-id), and requires mapping for compatibility reasons which should be considered when using [OpenTelemetry](/architecture/observability.md)
+- [Costs](https://aws.amazon.com/xray/pricing/) can escalate with increased usage
+- [OpenTelemetry](/architecture/observability.md) compatibility is not guaranteed due to `traceId` format differences from the [W3C specification](https://www.w3.org/TR/trace-context/#trace-id)
```

Comment:
PS: I have an action item to validate whether this is still accurate

Reply:
```suggestion
- **TODO** Validate that this is still accurate [OpenTelemetry](/architecture/observability.md) compatibility is not guaranteed due to `traceId` format differences from the [W3C specification](https://www.w3.org/TR/trace-context/#trace-id)
```